rcl/include/rcl/graph.h

```diff
@@ -524,6 +525,122 @@ rcl_count_subscribers(
   const char * topic_name,
   size_t * count);
 
+/// Returns a list of all publishers to a topic.
+/// Each element in the list will contain the node name, node namespace, topic type,
+/// gid and the qos profile of the publisher.
```

Comment:
nitpick: keep the brief to one sentence, i.e. move these last two lines into the doc body and don't use the `///` for them.

Reply:
Done.